hs_collection_resource/page_processors.py

```diff
@@ -13,20 +15,6 @@ def landing_page(request, page):
     content_model = page.get_content_model()
     edit_resource = page_processors.check_resource_mode(request)
 
-    user = request.user
-    if user.is_authenticated():
-        user_all_accessible_resource_list = get_my_resources_list(user)
```

Comment:
This query was being made in both view/edit modes, but only used in edit mode. The query takes a couple seconds, so the view is sped up by a couple of seconds by only making this query in edit mode.